### ISSUE TITLE
Case sensitive queries broken due to `ToLower` usage

### ISSUE THREAD

**Issue by RamblingCookieMonster** [bk-cs/PSFalcon#19]
Hi!

I could be wrong, but

(1) it appears Crowdstrike FQL is case sensitive (sigh), and
(2) in at least one command, Get-CsHostId, we take a filter and [make it lowercase](https://github.com/bk-cs/PSFalcon/blob/master/hosts/Get-CsHostId.psm1#L51), destroying any necessary casing for queries (EDIT: [a few other spots](https://github.com/bk-cs/PSFalcon/search?q=tolower&unscoped_q=tolower))

The practical effect:  I can't search on tags (not sure if this is just how we have it set up, or a Crowdstrike thing in general, but when I tag an agent, it shows up with a case sensitive prefix `SensorGroupingTags/$tag`, not just `$tag`, so I can't search for these at all)

Happy to submit a PR if you'd like!

Also:  Thanks for writing this!

**Comment by RamblingCookieMonster** [comment]
I see that you're redesigning this in a branch, my bad for missing that, and it looks like you already have this fixed (and are also encoding as needed) - feel free to close out this issue now (or wait until you release v2 so other folks will see it).

Thanks again, this is super helpful : )

**Comment by bk-cs** [comment]
Thank you for raising the issue! I appreciate your feedback.

This is indeed something that is fixed in v2, but it should be fixed in the present version, so I'll take a look at it ASAP.

**Comment by RamblingCookieMonster** [comment]
Cool!  fwiw - the changes in #20 worked for me, at least for Get-CsHostId.  Encoding might also be nice (I _think_ the `+` in queries breaks things), and it looks like you have the code already in your branch, but personally don't need it at the moment, and not sure how much you want to backport

cheers!

**Comment by bk-cs** [comment]
You're correct; the '+' (AND statement) in an FQL filter doesn't translate in PSFalcon without doing the encoding, which I discovered while working on 'v2'. 

I'm hoping to release v2 once I've got the rest of the commands updated, but it does change a lot of the command names, so I might put both 'v1' and 'v2' on the PSGallery for download.

I'll make a blanket change to remove .ToLower() from those filters for now. It looked like it would help users because it would avoid those case sensitive mistakes, but I didn't test searching with GroupingTags before I added it, so I appreciate you finding that!

I look forward to any additional feedback you're able to provide. Thanks for the help!
